src/components/App/App.jsx

```diff
@@ -90,14 +142,14 @@ class App extends React.Component {
   }
 
   savedFilter() {
-    ipcRenderer.send('title', 'Notifications')
+    ipcRenderer.send('title', 'Tasks')
```

Comment:
It'd be cool if we had changeView set the title based on the view to be displayed. That can come in the future, though.